applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -86,6 +86,7 @@ def GetDefaultParameters(cls):
             },
             "distance_reinitialization": "variational",
             "parallel_redistance_max_layers" : 25,
+            "parallel_redistance_preserve_interface" : false,
```

Comment:
Not for this PR, but this is becoming a complete mess as different functionalities defaults are mixed.
@ddiezrod we should consider (though it would be painful) discussing about the 2-fluid solver I/O.

Reply:
completely agree